content/en/docs/examples/aws-simple.md

```diff
@@ -0,0 +1,397 @@
+
+If you have made changes to the CAA code that affects the pod VM image and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/tree/main/src/cloud-api-adaptor/aws#create-pod-vm-ami) to build the pod VM AMI. Once image build is finished then export image id to the environment variable `PODVM_AMI_ID`.
+
+{{% /tab %}}
+
+{{< /tabpane >}}
+
+### CAA container image
+
+{{< tabpane text=true right=true persist=header >}}
+{{% tab header="**Versions**:" disabled=true /%}}
+
+{{% tab header="Last Release" %}}
+
+Export the following environment variable to use the latest release image of CAA:
+
+```bash
+export CAA_IMAGE="quay.io/confidential-containers/cloud-api-adaptor"
+export CAA_TAG="v${CAA_VERSION}-amd64"
+```
+
+{{% /tab %}}
+
+{{% tab header="Latest Build" %}}
+
+Export the following environment variable to use the image built by the CAA CI on each merge to main:
+
+```bash
+export CAA_IMAGE="quay.io/confidential-containers/cloud-api-adaptor"
+```
+
+Find an appropriate tag of pre-built image suitable to your needs [here](https://quay.io/repository/confidential-containers/cloud-api-adaptor?tab=tags&tag=latest).
+
+```bash
+export CAA_TAG=""
+```
+
+> **Caution**: You can also use the `latest` tag but it is **not** recommended, because of its lack of version control and potential for unpredictable updates, impacting stability and reproducibility in deployments.
+
+{{% /tab %}}
+
+{{% tab header="DIY" %}}
+
+If you have made changes to the CAA code and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/install/README.md#building-custom-cloud-api-adaptor-image) to build the container image. Once the image is built export the environment variables `CAA_IMAGE` and `CAA_TAG`.
+
+{{% /tab %}}
+
+{{< /tabpane >}}
+
+### Create the AWS credentials file
+
+```bash
+cat <<EOF > install/overlays/aws/aws-cred.env
+AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+EOF
+```
+
+> **Note**: The values should be without quotes
+
+### Select peer-pods machine type
+
+{{< tabpane text=true right=true persist=header >}}
+{{% tab header="AMD SEV-SNP" %}}
+
+```bash
+export PODVM_INSTANCE_TYPE="m6a.large"
+export DISABLECVM="false"
+```
+
+Find more AMD SEV-SNP machine types on [this](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-work.html) AWS documentation.
+
+{{% /tab %}}
+
+{{% tab header="Non-Confidential" %}}
+
+```bash
+export PODVM_INSTANCE_TYPE="t3.large"
+export DISABLECVM="true"
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+### Populate the `kustomization.yaml` file
+
+Run the following command to update the [`kustomization.yaml`](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/install/overlays/aws/kustomization.yaml) file:
+
+```yaml
+cat <<EOF > install/overlays/azure/kustomization.yaml
```

Comment:
Hi @bpradipt , I found this `azure`. Is this a typo?

Reply:
indeed.. Good catch @Xynnn007  I'll raise a PR to fix this.